lib/MusicBrainz/Script/JSONDump.pm

```diff
@@ -423,7 +433,7 @@ sub run {
     my $exit_code = $self->run_impl($c);
 
     $c->connector->disconnect;
-    rmtree($TMP_EXPORT_DIR);
+    rmtree($self->tmp_export_dir);
```

Comment:
I think this probably answers my previous question -- I was confusing `--tmp-dir` with `tmp_dir` from lib/MusicBrainz/Script/MBDump.pm

Reply:
Running `rmtree` on what is now a configurable directory (and now under the final volume) made me a bit uneasy, so I pushed a1cc4e8